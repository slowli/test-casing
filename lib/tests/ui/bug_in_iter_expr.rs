use test_casing::{cases, test_casing, TestCases};

const CASES: TestCases<i32> = cases!([1, 2]);

#[test_casing(2, CASS)]
fn tested_function(_arg: i32) {
    // Does nothing
}

fn main() {}
