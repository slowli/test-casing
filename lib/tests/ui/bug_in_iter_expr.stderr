error[E0425]: cannot find value `CASS` in this scope
 --> tests/ui/bug_in_iter_expr.rs:5:18
  |
3 | const CASES: TestCases<i32> = cases!([1, 2]);
  | --------------------------------------------- similarly named constant `CASES` defined here
4 |
5 | #[test_casing(2, CASS)]
  |                  ^^^^ help: a constant with a similar name exists: `CASES`
